website/static/js/meetings.js

```diff
@@ -44,12 +50,36 @@ function Meetings(data) {
                 },
                 {
                     data: 'count',
-                    sortInclude: true
+                    sortInclude: true,
+                    custom: function(){
+                        return m('span.text-center', item.data.count );
+                    }
                 },
                 {
-                    data: 'active', // Data field name
-                    sortInclude: true,
-                    custom: function() { return item.data.active ? 'Yes' : 'No'; }
+                    data: 'location',
+                    sortInclude : true,
+                    custom : function() {
+                        console.log(item.data.location);
+                        return item.data.location; }
+                },
+                {
+                    data: 'start_date', // Data field name
+                    sortInclude : true,
+                    custom: function() {
+                        if (item.data.start_date === null && item.data.end_date === null){
+                            return;
+                        }
+                        if (item.data.start_date === null) {
+                            return item.data.end_date;
+                        }
+                        if (item.data.end_date === null) {
+                            return item.data.start_date;
+                        }
+                        if (item.data.end_date === item.data.start_date) {
+                            return item.data.end_date;
+                        }
+                        return item.data.start_date + ' ~ ' + item.data.end_date;
```

Comment:
I think a `-` would look better than `~` here. The `~` makes me think that the dates are approximate for some reason.